src/components/Uploads.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import UploadField from './UploadField';
+
+class Uploads extends Component {
+  render() {
+    return(
+      <section>
+        <div className='grid-row margin-bottom-tiny'>
+          <div className='grid-item item-s-12'>
+            <h4 className='font-size-small font-bold'>{this.props.title}</h4>
+          </div>
+        </div>
+        <div className='grid-row margin-bottom-basic'>
+          {this.props.files.map( file => (
+            <div key={file.key} className='grid-item item-s-6 item-m-3'>
+              <img src={file.downloadURL} alt="" />
+              <button onClick={() => this.props.deleteFile(file)}>Eliminar</button>
+            </div>
+          ))}
+          { // Depending on the number of images and if mutiple uploads are enabled, show/hide the upload field
```

Comment:
is this the only check against multiple files for a single file context?

Reply:
More or less. Dropzone has a prop for multiple files that will/wont allow u to upload multiple files. This is, when the Select Files dialogs open it would/wouldn't allow u to select multiple files. 
Another check could happen in `<NoticiaForm />` where the handling of the upload happens, so even if the user somehow manages to fuckup and upload more than one file we could limit it there. Tbh I don't think that would happen.